features/reports/reports.feature

```diff
@@ -51,6 +51,12 @@ Feature: Correctly formatted reports
       9 total warnings
       """
 
+    Examples:
+      | option         |
+      |                |
+      | --sort-by none |
+      | --sort-by n    |
```

Comment:
At some point we may want to support default CLI arguments in the reek configuration (I think there's an issue for that). Then, it becomes necessary to be able to undo the `--sort-by smelliness`. The `n` bit is to show that OptionParser will accept unique prefixes for these parameters and these will be handled correctly. That bit is not really an important feature so I can take it out of the scenario.